Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonAboutToolkitForm.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
```

Comment:
These should not be needed

Reply:
Not sure why they are showing up here, as they were all removed in the latest commit?

Reply:
Just going through each change in order ....